### ISSUE TITLE
Manual Launch

### ISSUE THREAD

**Issue by R00T88** [DarthTon/Xenos#6]
I'm using x86 injector version to load a 32bit dll into AAPG games that is 32 bit too.
But when i select manual lauch it never fing AAGame.exe but i run the game.
This is the debug log:

2015-09-04 18:46:08 *NORMAL*     Started on Windows 10.0.0.10240 x64. Driver status: 0xC0000033
2015-09-04 18:46:56 *CRITICAL*   Injection initiated. Mode: 1, process type: 2, pid: 0, mmap flags: 0x4, erasePE: 0, unlink: 0, thread hijack: 0, init routine: '', init arg: ''
2015-09-04 18:46:56 *NORMAL*     Waiting on process D:\Programmi (x86)\Steam\steamapps\common\America's Army\AAPG\Binaries\Win32\AAGame.exe
2015-09-04 18:48:48 *WARNING*    Process wait canceled by user

I cancel the process after i saw that it not get it after 30sec of AAGame.exe running

This is the config file

<XenosConfig>
	<imagePath>Hack.dll</imagePath>
	<manualMapFlags>4</manualMapFlags>
	<procName>D:\Programmi (x86)\Steam\steamapps\common\America&apos;s Army\AAPG\Binaries\Win32\AAGame.exe</procName>
	<hijack>0</hijack>
	<unlink>0</unlink>
	<erasePE>0</erasePE>
	<close>1</close>
	<krnHandle>0</krnHandle>
	<processMode>2</processMode>
	<injectMode>1</injectMode>
	<delay>5000</delay>
	<period>0</period>
	<procCmdLine/>
	<initRoutine/>
	<initArgs/>
</XenosConfig>

i saw now that there 's a problem getting right path of it because of ' ( America's Army in debug but America&apos;s in config file )

Could be that?

Another thing, i select manual map injection type and select erasePE but in config file remain erasePE always 0 is it right?

Regards

**Comment by DarthTon** [comment]
Yes, most likely the path is the problem. You can use NTFS directory junction as a workaround till I fix this. Issue with a config could be a bug, I'll take a look.

**Comment by DarthTon** [comment]
The path is irrelevant. All that matters is process name, "AAGame.exe" in your case. Maybe injector fails to strip process name properly from full path, but I failed to reproduce it. ErasePE is irrelevant for manual map injection methods. There is separate flag in 'manualMapFlags' config entry for this.

**Comment by R00T88** [comment]
Got the same property with ShooterGame.exe.
i select the exe from path click inject.
Popup open saing waiting for ShooterGame.exe.
i lauch ShooterGame.exe but it never find it.
Thank for resporse on erasepe flag.
